src/components/web3/SelectableImageList.tsx

```diff
@@ -1,4 +1,5 @@
 import { Dispatch } from "react";
+import genericNFTImage from "../../images/nft.png";
```

Comment:
Can you change the name of this file to `no-nft-image.png` to make it a bit clearer?